pkg/core/native/native_test/management_test.go

```diff
@@ -180,12 +180,16 @@ func TestManagement_GenesisNativeState(t *testing.T) {
 }
 
 func TestManagement_NativeDeployUpdateNotifications(t *testing.T) {
-	const cockatriceHeight = 3
+	const (
+		cockatriceHeight = 3
+		echidnaHeight    = 4
```

Comment:
I guess Domovoi is 4 as well then?

Reply:
OK, we may extend this test with Domovoi-related notifications.